### PR TITLE
Convert SettingsInviteView to tailwind and extract some components

### PR DESCRIPTION
**Pull request by OceanOak**
No changelog

This is the following changes:

- Switch the SettingsInviteView to Tailwind.
- Extract submit button and email input components.
- Remove unused CSS.

**Before**
image

**After**
image

### CODE REVIEW

File: client/src/components/Settings/SettingsInviteView.res [hunk 1]
```diff
@@ -33,50 +35,43 @@ let view = (state: T.t): list<Html.html<AppTypes.msg>> => {
...
+        list{Icons.fontAwesome("spinner"), Html.h3(list{Attrs.class'(%twc("m-0 pl-2.5"))}, list{Html.text("Loading")})}
       } else {
-        list{Html.h3(list{}, list{Html.text("Send invite")})}
+        list{Html.h3(list{Attrs.class'(%twc("m-0"))}, list{Html.text("Send invite")})}
       }
 
-      Html.button(
-        list{
-          Attrs.class'("submit-btn"),
-          Html.Attributes.disabled(state.loading),
```
Comment (pbiggar) [inline client/src/components/Settings/SettingsInviteView.res:44]: Is this attribute in the new version? I don't see it

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looking good, thanks!

File: client/src/components/Settings/SettingsInviteView.res (hunk 1 above)
Comment (OceanOak) [reply client/src/components/Settings/SettingsInviteView.res:44]: Sorry, I missed it when creating the component. Please let me know if I should change anything in the submitBtn component or if there is a better way to do it; I am not sure about it.

Comment (pbiggar) [reply client/src/components/Settings/SettingsInviteView.res:44]: This is good for now. As a later step, you can look at making it more like the Contributor page and reusing some of the components and types for buttons and loading state, but that's not necessary for this PR.

Comment (pbiggar) [reply client/src/components/Settings/SettingsInviteView.res:44]: Does the spinner work here? You should maybe test it out with a tunnel against production to see.

Comment (OceanOak) [reply client/src/components/Settings/SettingsInviteView.res:44]: It seems to be working. I'll try to figure out how to use a tunnel to make sure.

dark-Dark-Google-Chrome-2022-11

Comment (pbiggar) [reply client/src/components/Settings/SettingsInviteView.res:44]: Adding a doc about tunnels would also be useful, right now we just have this: https://github.com/darklang/dark/blob/main/docs/running-against-production.md

Comment (pbiggar) [reply client/src/components/Settings/SettingsInviteView.res:44]: My personal setup:

- run node http-server using: `http-server backend/static/ -b --cors -g`
- tunnel using cloudflare tunnels (much harder to setup than ngrok, but also very reliable)

Comment (OceanOak) [reply client/src/components/Settings/SettingsInviteView.res:44]: This is the result of testing it with a tunnel against production.

ocean-Dark-Google-Chrome-2022-11

**Review by pbiggar — APPROVED** [review]
Great, thanks! Looks good.

Unfortunately a merge conflict crept in - if you can update to handle it then I can merge.

**Comment by pbiggar** [comment]
Great, thanks again.

**Pull request merged**